libs/blocks/section-metadata/section-metadata.js

```diff
@@ -1,19 +1,36 @@
 import { debounce } from '../../utils/utils.js';
 
 function handleBackground(div, section) {
-  const pic = div.querySelector('picture');
+  const pic = div.background.content.querySelector('picture');
   if (pic) {
     section.classList.add('has-background');
     pic.classList.add('section-background');
+    handleFocalpoint(pic, div.background.content);
```

Comment:
Was it discussed to scrape this info from the image alt text description at all?  `(top, left) | desc...` Similar to the image_link PR. 
I'm just thinking of other context for other potential extra text rows w/ images like photo credit.

Reply:
@ryanmparrish : Currently we only handling focal points on the background images. alt text description is not handled in this story.